weave-js/src/common/components/WandbLoader.tsx

```diff
@@ -86,3 +88,48 @@ export const TrackedWandbLoader = ({
 
   return <WandbLoader {...props} />;
 };
+
+export const TrackedWaveLoader = ({
+  captureException,
+  name,
+  profilingCb,
+  samplingRate = 0.1,
+  track,
+  onComplete,
+  onStart,
+  size,
+}: TrackedWandbLoaderProps & {
+  size: 'small' | 'huge';
+}) => {
+  useLifecycleProfiling(
```

Comment:
**question:** seems like contents of this call are copy-pasted from the existing tracked loader above. are there subtle differences i'm missing? could we pull this logic out and reuse for both comps (and any other future loaders)?